config/features.yml

```diff
@@ -37,8 +37,6 @@
   enabled: true
 - name: voice_recorder
   enabled: true
-- name: mobile_v2
```

Comment:
@muhsin-k  feature flag, once added, can't be removed from the yml file. We can update the flag for customers by running a migration instead.  if there is no checks in code based on this flag. then lets just write a comment over this key that its depricated.

Reply:
Makes sense.